<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergerWindow</class>
 <widget class="QMainWindow" name="MergerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDF Merger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="pdf_list_widget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>410</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Add pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Remove pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_folder_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Add all from folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output_button">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>200</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Select output folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="merge_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Merge pdfs</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_path_lineedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="current_path_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>160</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Current output path:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
